h/schemas/admin_group.py

```diff
@@ -35,6 +35,18 @@ def user_exists_validator(form, value):
     return user_exists_validator
 
 
+@colander.deferred
```

Comment:
See https://docs.pylonsproject.org/projects/colander/en/latest/binding.html for an explanation of the colander feature being used here.